std/csv.d

```diff
@@ -1155,6 +1171,14 @@ public:
                 _input.range.popFront();
         }
 
+        // Add this check to handle trailing newlines
+        if (_input.range.empty || 
+            (_input.range.front == '\n' || _input.range.front == '\r'))
```

Comment:
Docs should probably mention that ``\r`` cannot be a valid value to start a value with.